### PR TITLE
Explain why niscope test code uses persistent simulation

### PR DESCRIPTION
**Pull request by marcoskirsch**
It was not obvious to me, it will be much less obvious to someone who hasn't been involved in the project from day one.

### CODE REVIEW

File: src/niscope/system_tests/test_system_niscope.py [hunk 1]
```diff
@@ -8,7 +8,15 @@
 import tempfile
 
 
-# We need a lock file so multiple tests aren't hitting the simulated HW at the same time
+# There are system tests below that need either a PXI-5124 or a PXI-5142 instead of the PXIe-5164 we use everywhere else
+# because of specific capabilities on those models. Due to internal NI bug 969274, opening a simulated session to those models
+# sometimes fails. As a workaround, the nimi-bot VMs are configured with one simulated instrument of each kind respectively
```
Comment (sbethur) [inline src/niscope/system_tests/test_system_niscope.py]: I think we should qualify usages of "simulated" with the kind of simulation i.e. session-based vs. persistent.

File: src/niscope/system_tests/test_system_niscope.py [hunk 2]
```diff
@@ -8,7 +8,15 @@
...
+# named "5124" and "5142". If you want to run these tests on your own system, you will need to create these two simulated
+# instruments.
+# In addition, we need a global lock in order to keep us from opening more than one session to the same simulated instrument
```
Comment (sbethur) [inline src/niscope/system_tests/test_system_niscope.py]: Here again we should explicitly call out persistent simulation.

File: src/niscope/system_tests/test_system_niscope.py (hunk 1 above)
Comment (marcoskirsch) [reply src/niscope/system_tests/test_system_niscope.py]: Fixed in #1286

File: src/niscope/system_tests/test_system_niscope.py (hunk 2 above)
Comment (marcoskirsch) [reply src/niscope/system_tests/test_system_niscope.py]: Fixed in #1286

**Pull request merged**